static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,125 @@
-.email-composition-dialog {
-  .email-composition-contents {
-    display: flex;
-    flex-direction: column;
+.email-composition-dialog-wrapper {
```

Comment:
is the stuff here copied from your other PR? it would be good to just keep it to what we need to style the stuff in this PR I think